### DISCUSSION TITLE
Option outfmt 102

### DISCUSSION THREAD

**Discussion by CaroleBelliardo** [bbuchfink/diamond#438] [Ideas]
Hello, 
I use diamond with the option outfmt 102 and it is a really useful tool. But I think it would be really nice to add a columns about the total number of hits supporting the LCA taxonomy of each sequence, and also maybe the identity percent of the best hit. 
Thank you for your work that allows for amazing analysis.

**Comment by bbuchfink** [comment]
Hi Carole,
thanks for the suggestion. I do plan to extend the taxonomy format with some more information and will see to include this.
